src/solver/lps/LpsFromAntares.cpp

```diff
@@ -43,7 +45,8 @@ const WeeklyDataFromAntares& LpsFromAntares::weeklyData(WeeklyProblemId id) cons
     auto it = weeklyProblems.find(id);
     if (it == weeklyProblems.end())
     {
-        return WeeklyDataFromAntares(); // TODO Better error handling
+        throw Antares::Error::RuntimeError(
```

Comment:
You probably forgot to call `fmt::format` here. Don't forget to include it at the top.